README.md

```diff
@@ -67,22 +53,20 @@ The stamp descriptor properties are made available on each stamp as `stamp.compo
 * `methods` - A set of methods that will be added to the object's delegate prototype.
 * `properties` - A set of properties that will be added to new object instances by assignment.
 * `deepProperties` - A set of properties that will be added to new object instances by assignment with deep property merge.
-* `initializers` - A set of functions that will run in sequence and passed the data necessary to initialize a stamp instance.
+* `initializers` - A set of functions that will run in sequence. Stamp details and arguments get passed to initializers.
 * `staticProperties` - A set of static properties that will be copied by assignment to the stamp.
 * `propertyDescriptors` - A set of [object property
-descriptors](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperties)
-used for fine-grained control over object property behaviors
-* `configuration` - A set of options made available to the stamp and its initializers during object instance creation.
-Configuration properties get deep merged.
+descriptors](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperties) used for fine-grained control over object property behaviors.
+* `configuration` - A set of options made available to the stamp and its initializers during object instance creation. Configuration properties get deep merged.
 
 #### Composing descriptors
 
 Descriptors are composed together to create new descriptors with the following rules:
 
-* `methods` are shallow mixed: `descr.methods = _.assign({}, descr1.methods, descr2.methods)`
-* `properties` are shallow mixed: `descr.properties = _.assign({}, descr1.properties, descr2.properties)`
-* `deepProperties` are deep merged: `descr.deepProperties = _.merge({}, descr1.deepProperties, descr2.deepProperties)`
-* `initializers` are stacked: `descr.initializers = descr1.initializers.concat(descr1.initializers)`
-* `staticProperties` are shallow mixed: `descr.staticProperties = _.assign({}, descr1.staticProperties, descr2.staticProperties)`
-* `propertyDescriptors` are deep merged: `descr.propertyDescriptors = _.merge({}, descr1.propertyDescriptors, descr2.propertyDescriptors)`
-* `configuration` are deep merged: `descr.configuration = _.merge({}, descr1.configuration, descr2.configuration)`
+* `methods` are copied by assignment: `stamp.methods = _.assign({}, stamp1.methods, stamp2.methods)`
```

Comment:
`stamp` does not have `.methods`, `.properties`, etc. Only descriptor has those.
Stamp can have only `.compose`.

Confused yourself to stampit? :)